components/pages/dashboard/ads/AdsForm.tsx

```diff
@@ -0,0 +1,319 @@
+import clsx from 'clsx'
+import { Icon } from '@iconify/react'
+import { useRouter } from 'next/router'
+import { useForm } from 'react-hook-form'
+import { button } from '~components/Dashboard/Button'
+import Field from '~components/Dashboard/Form/Field'
+import { FileInput, Input } from '~components/Dashboard/Input'
+import { RadioGroup } from '~components/Dashboard/Radio'
+import Image from 'next/image'
+import { isAfter } from 'date-fns'
+
+type FormValue = {
+  adName: string
+  startDate: string
+  endDate: string
+  image: any
+  headline: string
+  description: string
+  landingUrl: string
+  callToActionUrl: string
+  discordPlacement: string
+  telegramPlacement: string
+  metaversePlacement: string
+}
+
+export function AdsForm() {
+  const { back } = useRouter()
+  const { handleSubmit, control, watch } = useForm<FormValue>({
+    defaultValues: {
+      discordPlacement: '',
```

Comment:
why not init all values?

Reply:
just for testing, removed. thanks for mentioning